.rubocop.yml

```diff
@@ -359,8 +359,7 @@ Layout/EmptyLinesAroundClassBody:
   Description: 'these are used to increase readability'
 
 Layout/EmptyLinesAroundMethodBody:
-  Enabled: false
-  Description: 'these are used to increase readability'
+  Enabled: true
```

Comment:
Without this, the following code:
```
def cleanup
  begin
    disconnect
  rescue StandardError
    nil
  end
end
```

Is automatically fixed to:
```
def cleanup

  disconnect
rescue StandardError
  nil

end
```

It's also not a bad rule to enable either